lib/rex/post/meterpreter/packet.rb

```diff
@@ -783,10 +783,10 @@ def aes_decrypt(key, iv, data)
   # the serialized TLV content, and then returns the key plus the
   # scrambled data as the payload.
   #
-  def to_r(session_guid=nil, key=nil)
+  def to_r(session_guid = nil, key = nil)
     xor_key = (rand(254) + 1).chr + (rand(254) + 1).chr + (rand(254) + 1).chr + (rand(254) + 1).chr
 
-    raw = [(session_guid || "0"*32).gsub(/-/, '')].pack('H*')
+    raw = [(session_guid || '00' * SESSION_GUID_SIZE).gsub(/-/, '')].pack('H*')
```

Comment:
I actually fixed this up in the packet pivot PR as well. The `session_guid` defaults to this when not specified. This looks fine though, will merge into packet pivot when landed.